### DISCUSSION TITLE
GTM Brainstorm

### DISCUSSION THREAD

**Discussion by XavierGeerinck** [dapr/js-sdk#280] [Ideas]
How do we want to promote the Dapr JS SDK?

Some ideas that could be checked:

* Ask "advocates" in the JS community what they think of Dapr (coming to mind: Kent C. Dodds)
* Speak on Conferences about the Dapr JS SDK

**Comment by shubham1172** [comment]
I think we almost created the same discussion in parallel 😅Closing #281
